SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+
+The Steering Committee is elected by the Project Members.
+
+#### Decision Making
+
+The Steering Committee should strive to make decisions by consensus.
+Where possible and appropriate, consensus should be reached between the members of the committee, and that consensus should reflect the consensus of the whole SunPy community.
+If consensus can not be reached between the committee members, decisions can be taken by a majority vote.
+
+#### Electing the Steering Committee
+
+An election to fill a vacant role on the Steering Committee will occur in two stages:
+
+* Nominations: No less than three weeks before an election a public announcement of the election must be made and nominations solicited for no less than two weeks.
+Candidates who have accepted their nomination (self nominations are allowed) should write a short statement supporting their candidacy, which will be made public as soon as possible, but no later than before the election starts.
+* Election: All candidates that have written statement will be put to a vote of the Project Members and the candidate with the most votes shall be elected to the position.
+In the event that more than one position has to be filled simultaneously each position shall be filled as if it were an independent election.
+
+#### Term
+
+Each position on the Steering Committee has a term of three years, from the date of the announcement of the election results.
+
+If there is a vacancy part way through a term, then a by-election will be held where the candidate elected to the committee will serve the remainder of the term of the member they are replacing.
+
+For the first election of the coordinating committee, one member will have a one year term, one member a two year term and one member a three year term.
+The term length of the elected candidates may be decided between themselves, in the case agreement can not be reached the person elected to the financial officer role will serve three years, the deputy lead developer two years and the lead developer one year.
+
+#### Removing Steering Committee or Advisory Board Members
+
+In exceptional circumstances, the Project Members may remove one or more sitting Steering Committee or Advisory Board members via a single vote.
+
+Such a vote is triggered when a Project Member calls for one publicly on an appropriate Project-wide communication channel, and two other active Project Members second the proposal.
+
+For the member(s) to be removed, two-thirds of the active Project Members (where the count includes the member being voted on if they are a Project Member) must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Project Member who has called for, or seconded such a vote, may not call for or second a vote to remove any member for one year from the original motion.
```

Comment:
```suggestion
To avoid repeated removal votes, any individual Project Member who has called for, or seconded such a vote, may not call for or second a vote to remove any member(s) for one year from the original motion.
```